### DISCUSSION TITLE
能否将窗口样式应用在非MainWindow上面呢？

### DISCUSSION THREAD

**Discussion by clear-zhao** [iNKORE-NET/UI.WPF.Modern#102] [Q&A] [answered]
我是wpf新手，就是想在项目里使用一下这个库，但是我项目已经写了好多了，无法在MainWindow里面使用了，我看文档第一章设置里，下载好库以后在App.xaml里合并了资源，然后就在MainWindow里面设置了，我想问能不能在其他窗口设置样式呢？
我尝试把<ui:ThemeResources/>和<ui:XamlControlsResources/>放在我想改变的那个window下面，但是页面样式并没有改变。

**Comment by NotYoojun** [comment, marked as answer]
ui:ThemeResources 和 ui:XamlControlsResources 都是 App.xaml 里的。每个窗口里面只需要设置 UseModernWindowStyle 为 True 就可以了。
